THEMES.md

```diff
@@ -517,6 +517,10 @@ Can be created as an extra.
 	- The image will be resized as large as possible so that it fits within this size and maintains its aspect ratio.  Use this instead of `size` when you don't know what kind of image you're using so it doesn't get grossly oversized on one axis (e.g. with a game's image metadata).
 * `origin` - type: NORMALIZED_PAIR.
 	- Where on the image `pos` refers to.  For example, an origin of `0.5 0.5` and a `pos` of `0.5 0.5` would place the image exactly in the middle of the screen.  If the "POSITION" and "SIZE" attributes are themable, "ORIGIN" is implied.
+* `rotation` - type: FLOAT.
```

Comment:
These changes are the result of an[ earlier commit](https://github.com/RetroPie/EmulationStation/commit/2377175a1995ae4cf707bce90005c14028db75ed), did you rebase your branch before squashing your commits, or did you merge in these changes manually? This way, we are reviewing those changes as well, while they are already part of the code-base.